Lib/test/test_os.py

```diff
@@ -1252,6 +1253,43 @@ def tearDownClass(cls):
         os.rmdir(support.TESTFN)
 
 
+class CurrentDirTests(unittest.TestCase):
+
+    def setUp(self):
+        base = os.path.abspath(support.TESTFN)
+        self.tmp_dir = base + '_dir'
+        self.tmp_lnk = base + '_lnk'
+
+    def test_getcwd(self):
+        # os.getcwd() always returns the dereferenced path
+        with support.temp_cwd(self.tmp_dir):
+            os.chdir(self.tmp_dir)
+            self.assertEqual(self.tmp_dir, os.getcwd())
```

Comment:
Swap arguments. The first argument is an actual value, the second argument is an expected value.